alphabet-Order-task.js

```diff
@@ -0,0 +1,11 @@
+// Napisz funkcję alphabetOrder(), która zwróci podany łańcuch znaków z literami
+// uporządkowanymi według alfabetu („wyraz” - „arwyz”). Nie przejmuj się na razie polskimi
+// znakami :)
+
+function alphabetOrder(word) {
```

Comment:
`word` is not a good name for this parameter. It doesn't have to be a word. I can be everything. Word, sentence, empty string etc.